ArduPlane/events.cpp

```diff
@@ -172,6 +172,10 @@ void Plane::handle_battery_failsafe(const char *type_str, const int8_t action)
             afs.gcs_terminate(true, battery_type_str);
             break;
 
+        case Failsafe_Action_Parachute:
+            parachute_release();
```

Comment:
Should the vehicle be disarmed here? I haven't investigated the parachute code in the past, but what keeps us from spinning motors once the parachute has been released?

Reply:
The throttle is suppressed when the parachute release is initiated.

https://github.com/ArduPilot/ardupilot/blob/12e62d010760c8b98af98a5c973740a8976405db/ArduPlane/servos.cpp#L54-L62